### PR TITLE
Added flagset support to factory

### PR DESCRIPTION
**Pull request by chillaq**
# Python SDK

### CODE REVIEW

File: splitio/client/factory.py [hunk 1]
```diff
@@ -350,7 +350,7 @@ def _build_in_memory_factory(api_key, cfg, sdk_url=None, events_url=None,  # pyl
     }
 
     storages = {
-        'splits': InMemorySplitStorage(),
+        'splits': InMemorySplitStorage(cfg['FlagSetsFilter'] if cfg['FlagSetsFilter'] is not None else []),
```
Comment (mmelograno) [inline splitio/client/factory.py:353]: we should be consistent with cfg and use first letter lowercase

**Review by mmelograno — CHANGES_REQUESTED** [review]

File: splitio/client/factory.py (hunk 1 above)
Comment (chillaq) [reply splitio/client/factory.py]: will fix

File: splitio/models/telemetry.py [hunk 2]
```diff
@@ -41,6 +41,7 @@ class ConfigParams(Enum):
     EVENTS_QUEUE_SIZE = 'eventsQueueSize'
     IMPRESSIONS_MODE = 'impressionsMode'
     IMPRESSIONS_LISTENER = 'impressionListener'
+    FLAG_SETS = 'FlagSetsFilter'
```
Comment (mmelograno) [inline splitio/models/telemetry.py:44]: should start with lowecase?

**Review by mmelograno — CHANGES_REQUESTED** [review]

File: splitio/models/telemetry.py (hunk 2 above)
Comment (chillaq) [reply splitio/models/telemetry.py:44]: Correct,  the class (ConfigParams) should be lower case, will check and correct all the constant classes in Telemetry model.

File: splitio/models/telemetry.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ class CounterConstants(Enum):
     EVENTS_QUEUED = 'eventsQueued'
     EVENTS_DROPPED = 'eventsDropped'
 
-class ConfigParams(Enum):
+class _ConfigParams(Enum):
```
Comment (mmelograno) [inline splitio/models/telemetry.py:30]: why classes with `_` at the begining?

**Review by mmelograno — CHANGES_REQUESTED** [review]

File: splitio/models/telemetry.py (hunk 3 above)
Comment (chillaq) [reply splitio/models/telemetry.py:30]: The constants class is used only internal in telemetry model.

**Pull request merged**